pkgs/servers/dns/doh-proxy-rust/cargo-lock.patch

```diff
@@ -0,0 +1,838 @@
+diff --git a/Cargo.lock b/Cargo.lock
```

Comment:
It would be great if you could try to upstream this.

Reply:
Unfortunately, upstream has rejected this in the past: https://github.com/jedisct1/doh-server/issues/31